src/Context.fs

```diff
@@ -44,11 +44,11 @@ and HttpRequest = {
     /// Responsetype. JSON or Protobuf
     ResponseType: ResponseType
     /// List of headers to be sent
-    Headers: (string * string) list
+    Headers: Map<string, string>
```

Comment:
The comment still says "list of headers", don't know if that's something you want to change?

Reply:
Thanks, fixed.